meta/ee-requirements.txt

```diff
@@ -0,0 +1,7 @@
+# ansible-builder doesn't seem to properly handle "; python_version" type of constraints
+# requirements here are assuming python 3.6 or higher
+hvac >=0.10.6
+urllib3 >= 1.15
+
+boto3       # these are only needed if inferring AWS credentials or
+botocore    # using a boto profile; including for completeness
```

Comment:
Is there a reason why these requirements are not in the `requirements.txt` file at the collection's root? This way, we do not need the `meta/execution-environment.yml` file at all since the `requirements.txt` file is automatically used by ansible-builder as a source of dependencies.

Reply:
@tadeboro that's what @erinn had originally proposed; there currently isn't one in the root. I wanted to point at the one used by the integration/unit tests since it already existed, but I found that builder doesn't handle constraints correctly, like this:
```
hvac >= 0.10.6, != 0.10.12, != 0.10.13, < 1.0.0 ; python_version == '2.7' # bugs in 0.10.12 and 0.10.13 prevent it from working in Python 2
hvac >= 0.10.6 ; python_version >= '3.5'
```

Resulted in a single line like this:
```
hvac >=0.10.6, >= 0.10.6, != 0.10.12, != 0.10.13, < 1.0.0 
```

So I opted to make a separate one for this purpose for the time being that assumes py >= 3.6

Reply:
IIRC, EEs use Python 3.8, so those markers are usually not needed. But maybe builder should learn how to deal with them? Anyhow, this PR looks good and I think it is ready to go.

Reply:
@tadeboro so interestingly enough, I suspect it might be the case that some standard version or range of versions was used for EE's, and I tried to verify it but could not find it in the docs. If you happen to know where that is, I would really appreciate a link!

In any case, thanks again for reviewing.